examples/canvas.rs

```diff
@@ -42,49 +42,48 @@ fn main() {
 
 // Draw the Ui.
 fn set_widgets(ui: &mut Ui) {
-    use conrod::color::{blue, light_orange, orange, dark_orange, red, white};
     use conrod::{Button, Colorable, Labelable, Positionable, Sizeable, Tabs, Text, WidgetMatrix};
 
     // Construct our main `Canvas` tree.
     Canvas::new().flow_down(&[
-        (HEADER, Canvas::new().color(blue()).pad_bottom(20.0)),
+        (HEADER, Canvas::new().color(::conrod::color::BLUE).pad_bottom(20.0)),
```

Comment:
If possible, could you import the `color` module and instantiate these colors like `color::BLUE` instead of `::conrod::color::BLUE`?